zeroex/orderwatch/order_watcher_test.go

```diff
@@ -933,6 +933,56 @@ func TestOrderWatcherBatchEmitsAddedEvents(t *testing.T) {
 	require.Len(t, orders, 2)
 }
 
+func TestOrderWatcherCleanup(t *testing.T) {
+	if !serialTestsEnabled {
+		t.Skip("Serial tests (tests which cannot run in parallel) are disabled. You can enable them with the --serial flag")
+	}
+
+	teardownSubTest := setupSubTest(t)
+	defer teardownSubTest(t)
+
+	meshDB, err := meshdb.New("/tmp/leveldb_testing/" + uuid.New().String())
+	require.NoError(t, err)
+
+	ctx, cancelFn := context.WithCancel(context.Background())
+	defer cancelFn()
+	blockWatcher, orderWatcher := setupOrderWatcher(ctx, t, ethRPCClient, meshDB)
+
+	// Create and add two orders to OrderWatcher
+	amount := big.NewInt(10000)
+	signedOrderOne := scenario.CreateZRXForWETHSignedTestOrder(t, ethClient, makerAddress, takerAddress, amount, amount)
+	watchOrder(ctx, t, orderWatcher, blockWatcher, ethClient, signedOrderOne)
+	signedOrderTwo := scenario.CreateZRXForWETHSignedTestOrder(t, ethClient, makerAddress, takerAddress, amount, amount)
+	watchOrder(ctx, t, orderWatcher, blockWatcher, ethClient, signedOrderTwo)
+	signedOrderOneHash, err := signedOrderTwo.ComputeOrderHash()
+	require.NoError(t, err)
+
+	// Set lastUpdate for signedOrderOne to more than defaultLastUpdatedBuffer so that signedOrderOne
+	// does not get re-validated by the cleanup job
+	signedOrderOneDB := &meshdb.Order{}
+	err = meshDB.Orders.FindByID(signedOrderOneHash.Bytes(), signedOrderOneDB)
+	require.NoError(t, err)
+	signedOrderOneDB.LastUpdated = time.Now().UTC().Add(-defaultLastUpdatedBuffer).Add(-1 * time.Minute)
```

Comment:
nit: You don't need to convert to UTC here since we do it within the `meshdb` package (and even that only matters for indexes/sorting. You don't need to convert to UTC in general.). We also don't need two separate calls to `Add`. How about:

```go
time.Now().Add(-defaultLastUpdatedBuffer - 1 * time.Minute)
```